Microsoft.Toolkit.Uwp.UI/Shadows/Shadows.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI
+{
+    /// <summary>
+    /// Helper class for attaching <see cref="AttachedShadowBase"/> shadows to <see cref="FrameworkElement"/>s.
+    /// </summary>
+    public static class Shadows
```

Comment:
Community Suggested we call this `Effects` and have the property be called `Shadow` so we'd get:
`<ui:Effects.Shadow>` for the attached property.